src/Microsoft.Azure.WebJobs.Extensions.Kafka/Serialization/LocalSchemaRegistry.cs

```diff
@@ -43,26 +43,37 @@ public Task<List<string>> GetAllSubjectsAsync()
             throw new System.NotImplementedException();
         }
 
-        public Task<Schema> GetLatestSchemaAsync(string subject)
+        public Task<RegisteredSchema> GetLatestSchemaAsync(string subject)
         {
             throw new System.NotImplementedException();
         }
 
-        public Task<string> GetSchemaAsync(int id)
+        public Task<RegisteredSchema> GetRegisteredSchemaAsync(string subject, int version)
         {
-            return Task.FromResult(this.schema);
+            throw new System.NotImplementedException();
         }
 
         public Task<string> GetSchemaAsync(string subject, int version)
```

Comment:
Is this method still part of the interface? If not, you can remove it.

Reply:
Yes. It is Obsolete, though. :) 
![image](https://user-images.githubusercontent.com/1390976/86269064-10c76d80-bb7e-11ea-8d04-767183ecbb41.png)

Reply:
Which ones are obsolete?